Firmware/MotorControl/current_limiter.cpp

```diff
@@ -0,0 +1,84 @@
+
+#include "odrive_main.h"
+
+#include <algorithm>
+
+#include "gpio.h"
+#include "low_level.h"
+
+ThermistorCurrentLimiter::ThermistorCurrentLimiter(const Config_t& config) :
+        config_(config),
+        error_(ERROR_NONE)
+{
+    // For onboard thermistors the gpio_pin is the direct ADC channel
+    // because it's connected to a GPIO that's not exposed to the user.
+    // We set the gpio_pin_ to UINT16_MAX so it's clear that this
+    // value is not a "real" gpio pin.
+    if (config_.onboard)
+    {
```

Comment:
Our convention is to keep braces on the same line. See [here](https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions) and [here](https://google.github.io/styleguide/cppguide.html#Conditionals).

Reply:
Will fix.